src/main/java/io/appform/dropwizard/actors/actor/hierarchical/HierarchicalUnmanagedBaseActor.java

```diff
@@ -0,0 +1,172 @@
+ * This is not Managed and will not be automatically started by dropwizard.</p>
+ */
+@Data
+@EqualsAndHashCode
+@ToString
+@Slf4j
+public class HierarchicalUnmanagedBaseActor<MessageType extends Enum<MessageType>, Message> {
+
+    private final HierarchicalTreeConfig<HierarchicalActorConfig, String, HierarchicalSubActorConfig> hierarchicalTreeConfig;
+    private final ConnectionRegistry connectionRegistry;
+    private final ObjectMapper mapper;
+    private final RetryStrategyFactory retryStrategyFactory;
+    private final ExceptionHandlingFactory exceptionHandlingFactory;
+    private final Class<? extends Message> clazz;
+    private final Set<Class<?>> droppedExceptionTypes;
+    private final MessageType messageType;
+    private final MessageHandlingFunction<Message, Boolean> handlerFunction;
+    private final MessageHandlingFunction<Message, Boolean> expiredMessageHandlingFunction;
+
+    private HierarchicalDataStoreSupplierTree<
+            HierarchicalSubActorConfig,
+                        HierarchicalActorConfig,
+                        MessageType,
+            Actor<MessageType, ? extends Message>> worker;
+
+
+    public HierarchicalUnmanagedBaseActor(MessageType messageType,
+                                          HierarchicalActorConfig hierarchicalActorConfig,
+                                          ConnectionRegistry connectionRegistry,
+                                          ObjectMapper mapper,
+                                          RetryStrategyFactory retryStrategyFactory,
+                                          ExceptionHandlingFactory exceptionHandlingFactory,
+                                          Class<? extends Message> clazz,
+                                          Set<Class<?>> droppedExceptionTypes,
+                                          MessageHandlingFunction<Message, Boolean> handlerFunction,
+                                          MessageHandlingFunction<Message, Boolean> expiredMessageHandlingFunction) {
+        this.messageType = messageType;
+        this.hierarchicalTreeConfig = new HierarchicalTreeConfig<>(hierarchicalActorConfig, hierarchicalActorConfig.getChildren());
+        this.connectionRegistry = connectionRegistry;
+        this.mapper = mapper;
+        this.retryStrategyFactory = retryStrategyFactory;
+        this.exceptionHandlingFactory = exceptionHandlingFactory;
+        this.clazz = clazz;
+        this.droppedExceptionTypes = droppedExceptionTypes;
+        this.handlerFunction = handlerFunction;
+        this.expiredMessageHandlingFunction = expiredMessageHandlingFunction;
+    }
+
+    public void start() throws Exception {
+        log.info("Initializing Router");
+        this.initializeRouter();
+        worker.traverse(hierarchicalSubActor -> {
+            try {
+                hierarchicalSubActor.start();
+            } catch (Exception e) {
+                log.error("Unable to start worker: {}", hierarchicalSubActor);
```

Comment:
Whole actor is being logged. Should we look at logging only actor type or actor configuration (barring sensitive fields) ? We will have to make changes in every method in this class